hudi-cli/src/main/java/org/apache/hudi/cli/commands/ArchivedCommitsCommand.java

```diff
@@ -138,9 +139,11 @@ public String showCommits(
       throws IOException {
 
     System.out.println("===============> Showing only " + limit + " archived commits <===============");
-    String basePath = HoodieCLI.getTableMetaClient().getBasePath();
+    HoodieTableMetaClient metaClient = HoodieCLI.getTableMetaClient();
+    String basePath = metaClient.getBasePath();
+    Path archivePath = new Path(metaClient.getArchivePath() + "/.commits_.archive*");
     FileStatus[] fsStatuses =
-        FSUtils.getFs(basePath, HoodieCLI.conf).globStatus(new Path(basePath + "/.hoodie/.commits_.archive*"));
+        FSUtils.getFs(basePath, HoodieCLI.conf).globStatus(archivePath);
```

Comment:
This can affect all the old tables read archives.

Reply:
archivePath = new Path(metaClient.getArchivePath() + "/.commits_.archive*") is equivalent of new Path(basePath + "/.hoodie/.commits_.archive*"). the metaClient.getArchivePath() should return `basePath + "/.hoodie"` for all old tables.

Reply:
@n3nash, they are different,
one: `/table/.hoodie/archived/.commits_.archive*`
another: `/table/.hoodie/.commits_archive*`

Reply:
The right way is add `archiveFolderPattern` to `show archived commits command`
like `show archived commit stats` dose.

Reply:
@hmatu It return `/table/.hoodie/.commits_archive*` if old tables use `DEFAULT` `''` and return `/table/.hoodie/archived/.commits_.archive*` if old tables use path `archived` to archive. So it will return the correct path with archive path stored in .hoodie.
On the other hand，'archiveFolderPattern' `allow for users to be able to provide a full path of the files under the archive folder and read it`.

Reply:
Thanks for your explain, but I still think it's a right way that add `archiveFolderPattern` to `show archived commits command` like `show archived commit stats` dose.

Reply:
@hmatu I think this is just a clean up of code to remove the hard-coded "archived" and providing it through the archive folder name, this is fine IMO since this does not break any backwards compatibility @hddong confirm ?
What you are referring to is also correct, but that should be another PR to allow for reading a path pattern for `show archived commits` just like others have.

Reply:
@n3nash, this pr aims to "Adjust the read and write path of archive". IMO, it is unnecessary to create an new pr.

Reply:
@hmatu I agree, I think this PR is just doing some basic cleanup so we should just rename this PR and merge it and fix the adjusting of the read/write path in a different one. 
@hddong please rename this PR as "archived commits command code cleanup" and we can merge this